packages/xo-web/src/xo-app/xoa/licenses/xostor.js

```diff
@@ -24,40 +25,71 @@ class XostorLicensesForm extends Component {
     return bindLicense(this.state.licenseId, item.uuid).then(userData.updateLicenses)
   }
 
+  getAlerts = createSelector(
+    () => this.props.item,
+    () => this.props.userData,
+    (sr, userData) => {
+      const alerts = []
+      const licenses = userData.licensesByXostorUuid[sr.id]
+
+      // Xostor bound to multiple licenses
+      if (licenses?.length > 1) {
+        alerts.push({
+          level: 'danger',
+          render: (
+            <p>
+              {licenses.map(license => license.id.slice(-4)).join(',')} {_('xostorMultipleLicenses')}
```

Comment:
It looks bit weird with the IDs prepended to the message.
```suggestion
              {_('xostorMultipleLicenses')}<br />
              {licenses.map(license => license.id.slice(-4)).join(', ')}
```